LR/07/src/tcpserver.cpp

```diff
@@ -42,19 +42,31 @@ void TcpServer::slotClientDisconnected()
 }
 
 /**
- * @brief Метод отправляет 1 или 0 клиенту
- * @param int answer - ответ, отправляемый клиенту(1 или 0)
+ * @brief Метод отправляет клиенту строку в формате
+ * @param unsigned char grade - оценка за лабораторную работу
+ * @param QString comment - описание системной ошибки, либо комментарий сдающему лабораторную
  * @return void
  */
-void TcpServer::sendToClient(bool answer)
+void TcpServer::sendToClient(unsigned char grade, QString comment)
 {
-   if (answer) {
-       mTcpSocket->readAll();
-       mTcpSocket->write("1");
-   } else {
-       mTcpSocket->readAll();
-       mTcpSocket->write("0");
-   }
+    const unsigned char MESSAGE_TYPE = 2;
+    QJsonObject json;
+
+    if (comment != NULL) {
+        json ["massageType"] = MESSAGE_TYPE;
```

Comment:
Очепятка - ключ в json должен быть: messageType
```suggestion
        json ["messageType"] = MESSAGE_TYPE;
```